pom.xml

```diff
@@ -3965,6 +3972,22 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk15</id>
```

Comment:
Is that for jdk15 only? If not possibly name it `<id>jdk9+</id>`

Reply:
Thanks, renamed.